packages/grafana-i18n/tsconfig.json

```diff
@@ -8,5 +8,5 @@
   },
   "exclude": ["dist/**/*"],
   "extends": "@grafana/tsconfig",
-  "include": ["src/**/*.ts*", "../../public/app/types/*.d.ts"]
+  "include": ["src/**/*.ts*"]
```

Comment:
As we have the global types/overrides for `Intl` in this package, I don't think we need to include the app types

Reply:
Great catch 🙌